src/menu.js

```diff
@@ -61,7 +61,7 @@ function menuTempl() {
       },
       {
         label: 'Close',
-        accelerator: 'CommandOrControl+W',
+        accelerator: 'CommandOrControl+Q',
```

Comment:
Maybe we should think also about our friends using a Windows. I found something like in the electron docs:

```
      {
        label: 'Close',
        accelerator: (function() {
          if (process.platform == 'darwin')
            return 'CommandOrControl+Q';
          else
            return 'Alt+F4';
        })(),
        role: 'close'
      }
```